spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
```

Comment:
I'm not sure about this approach. Seeing as the interface is fixed to port 5080, wouldn't it make sense to just use that as the url?